.github/pull_request_template.md

```diff
@@ -37,6 +37,12 @@ If possible, please also clarify if this is a user-facing change compared to the
 If no, write 'No'.
 -->
 
+### Is the change a dependency upgrade?
+<!--
+If yes, please attach a file with output from mvn dependency:tree to validate a complete upgrade of dependency.
```

Comment:
Looks good but what would have been even cooler is to add a GitHub action to do that automatically ;)

Reply:
Great idea!! , will add that in a separate ticket :) This also gives contributors a chance to verify before pushing if just version change is enough, sometimes we miss that :)